packages/framework-integration-tests/integration/providers/aws/functionality/events.integration.ts

```diff
@@ -64,7 +65,7 @@ describe('events', async () => {
     )
 
     const mockCartId = random.uuid()
-    const eventsToCreate = 26 // Using 26 because batches are of 25, making sure the batching gets triggered
+    const eventsToCreate = 30 // Using 30 because batches are of 25, making sure the batching gets triggered
```

Comment:
Why 30 and not 26?

Reply:
I changed it because I'm a bit paranoid about the "limit" conditions. I wanted to be sure two batches were needed. If we use 26 and mess a bit in the loop where we register events (Imagine we start the loop in 1 instead of 0) then we would get here a false positive.

In brief, I wanted to have more than just 1 unit above the limit condition

Reply:
I see, agree 100%.

PS: That's why loops are evil :trollface: